assets/scripts/script.js

```diff
@@ -1,7 +1,41 @@
+//NAV SCRIPTS ----------------------------------------------------------------
 const burger = document.querySelector('.burger');
 const navElements = document.querySelector('#navbar-items');
 
 burger.addEventListener('click', function(){
     burger.classList.toggle('is-active');
     navElements.classList.toggle('is-active');
 })
+
+
+//MODAL SCRIPTS ----------------------------------------------------------------
+const modals = document.querySelectorAll('.modal');
+const modalTriggers = document.querySelectorAll('.staff-btn');
+const modalClose = document.querySelectorAll('.modal-close');
+const modalBackgrounds = document.querySelectorAll(".modal-background")
+
+for (let i = 0; i < modals.length; i++) {
```

Comment:
Can we put all this modal JS in its own file so we only import it in the Staff page where it is needed?